### DISCUSSION TITLE
Messages

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1349] [General]
I get those messages when opening MyTourbook
![2024-05-16 20_53_18-Comparateur de parcours - MyTourbook - 24 1 0](https://github.com/mytourbook/mytourbook/assets/4082252/a8ffb96f-ecfc-42f8-a4fc-0654f6c47339)

![2024-05-16 20_54_13-Comparateur de parcours - MyTourbook - 24 1 0](https://github.com/mytourbook/mytourbook/assets/4082252/c333fd62-52d9-4b4f-bb2e-03b7fd05b3d0)

**Comment by wolfgang-ch** [comment]
Attach the eclipse log file
